python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -826,6 +829,36 @@ def do_evaluate(
                     plc.interop.from_arrow(pa.scalar(stop, type=pa.int32())),
                 )
             )
+        elif self.name in {
+            pl_expr.StringFunction.StripChars,
+            pl_expr.StringFunction.StripCharsStart,
+            pl_expr.StringFunction.StripCharsEnd,
+        }:
+            column = self.children[0].evaluate(df, context=context, mapping=mapping)
+            expr_chars = self.children[1]
+            assert isinstance(expr_chars, Literal)
```

Comment:
This pre-condition needs to be checked in `__init__`. Since polars supports the chars to strip being a column.

Can you also open a feature request to support stripping characters with per-row-specification (i.e. a `column_view` of `to_strip`).